subgraph-js/src/types.ts

```diff
@@ -95,8 +96,9 @@ export function entitiesResolver({
       }
     }
 
+    const extensions = type.extensions as ApolloGraphQLObjectTypeExtensions;
```

Comment:
Rather than casting, just declare the type without the Readonly.

```suggestion
    const extensions: ApolloGraphQLObjectTypeExtensions = type.extensions;
```